src/pilot/config_idrac.py

```diff
@@ -405,7 +412,8 @@ def config_idrac(instack_lock,
     pxe_nic_fqdd = get_pxe_nic_fqdd(
         pxe_nic,
         model_properties,
-        drac_client)
+        ip_service_tag,
```

Comment:
I don't understand why it's necessary to change the method signature here.

Reply:
Answered this on line 72